packages/experiments/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@wordpress/dependency-injection",
+	"name": "@wordpress/experiments",
```

Comment:
I see that line 13 and 17 need to get updated.

In the keywords section we should replace `dom` and `utils` with `experiments` or something like that.